sys/linux/auto.txt

```diff
@@ -1109,13 +1109,13 @@ act_bpf_policy$auto_act_bpf [
 	TCA_ACT_BPF_PARMS	nlattr[TCA_ACT_BPF_PARMS, intptr]
 	TCA_ACT_BPF_OPS_LEN	nlattr[TCA_ACT_BPF_OPS_LEN, int16]
 	TCA_ACT_BPF_OPS		nlattr[TCA_ACT_BPF_OPS, int16]
-	TCA_ACT_BPF_FD		nlattr[TCA_ACT_BPF_FD, int32]
+	TCA_ACT_BPF_FD		nlattr[TCA_ACT_BPF_FD, fd]
 	TCA_ACT_BPF_NAME	nlattr[TCA_ACT_BPF_NAME, string]
 ] [varlen]
 
 allowedip_policy$auto_netlink [
 	WGALLOWEDIP_A_FAMILY	nlattr[WGALLOWEDIP_A_FAMILY, int16]
-	WGALLOWEDIP_A_IPADDR	nlattr[WGALLOWEDIP_A_IPADDR, intptr]
+	WGALLOWEDIP_A_IPADDR	nlattr[WGALLOWEDIP_A_IPADDR, int16]
```

Comment:
Is it really 2 bytes long?
Should be 4 bytes: https://elixir.bootlin.com/linux/v6.10.7/source/drivers/net/wireguard/netlink.c#L48

Also, maybe we'd better use `ipv4_addr` here?

Reply:
It's now an array[int8]